mungegithub/github/github.go

```diff
@@ -539,6 +540,26 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 	return nil
 }
 
+// Priority returns the priority an issue was labeled with.
+// The labels must take the form 'priority/P?[0-9]+'
+// or math.MaxInt32 if unset
+func (obj *MungeObject) Priority() int {
+	priority := math.MaxInt32
```

Comment:
I really don't like that idea.  I don't want to punish someone for taking the time to approprietely prioritize something. I'd rather 'punish' the people who are lazy and provide no transparency...